.github/workflows/backport.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Backport Assistant Runner
+
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    if: github.event.pull_request.merged
```

Comment:
Does this negate the `labeled` trigger above? It'd be sweet if we could label a PR that's already been merged and have the backport run

Reply:
On Consul core this is possible by adding the label and manually re-running the CI job, which is super-unintuitive to find, would love if this could happen from just adding the label post-merge.

Reply:
I /think/ this is only checking if the pr has actually been merged before it runs the backport runner, since the merged value is a bool yes/no, which I would argue is probably desirable in that we don't want this to run on PRs that haven't merged yet. If I'm reading everything right, it should run on closed and on labeled, but only run on labeled if the pr has already been merged.

Reply:
I can't find a good docs source describing `.merged`; however, from the examples I see in various places, I agree with you @sarahalsmiller